nidirect_news/nidirect_news.module

```diff
@@ -35,10 +35,21 @@ function nidirect_news_preprocess_field(array &$variables) {
  * Implements hook_preprocess_node().
  */
 function nidirect_news_preprocess_node(array &$variables) {
-  if ($variables['node']->getType() == 'featured_content_list' && $variables['view_mode'] == 'full') {
-    // Replace the node view with a rendered view display, using the node items as arguments.
-    // This mirrors the display you would see on the embedded views display.
-    $variables['content'] = \Drupal::service('nidirect_news.news')->getFeaturedNews();
+  if ($variables['node']->getType() != 'featured_content_list') {
+    return;
+  }
+
+  if ($variables['view_mode'] == 'full' && !empty($variables['node']->field_tags->target_id)) {
+    $news_tag = \Drupal::entityTypeManager()
+      ->getStorage('taxonomy_term')
+      ->loadByProperties(['name' => 'News']);
+    $news_tag = reset($news_tag);
+
+    if ($variables['node']->field_tags->target_id == $news_tag->id()) {
```

Comment:
Crucially, this was missing before. Without it, it swaps the homepage FCL render array around as well which is not what we want!